src/content/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-apprunner.mdx

```diff
@@ -0,0 +1,163 @@
+
+[Default polling information](/docs/infrastructure/amazon-integrations/aws-integrations-list/aws-polling-intervals-infrastructure-integrations) for the AWS integrations:
+- New Relic polling interval: 5 minutes
+- Amazon CloudWatch data interval: 1 minute or 5 minutes
+
+
+## Find and use data
+To find your integration data, go to your account on [one.newrelic.com ](one.newrelic.com).
+
+For logs:
+On the left column select **Logs > Attributes > AWS** and select **logGroup** and/or **logStream**
+
+For metrics:
+On the left column select **Data explorer > Metrics** and filter by `aws.apprunner`
+
+## Metric data
+
+<table>
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+      Metric (min, max, average)  
+      </th>
+      <th>
+      Unit  
+      </th>
+      <th>
+      Description
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+      `2xxStatusResponses`
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Number of successfull requests to the service
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `4xxStatusResponses`  
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Number of failed requests to the service
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `ActiveInstances`
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Number of active instances
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `CPUUtilization`
+      </td>
+      <td>
+      percentage  
+      </td>
+      <td>
+      Instance CPU loading
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `MemoryUtilization`
+      </td>
+      <td>
+      percentage  
+      </td>
+      <td>
+      Instance memory utilization
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `RequestLatency`
+      </td>
+      <td>
+      ms  
+      </td>
+      <td>
+      How long a request takes
+      </td>
+    </tr>
+    <tr>
+      <td>
+      `Requests`
+      </td>
+      <td>
+      count  
+      </td>
+      <td>
+      Total number of requests
+      </td>
+    </tr>
+    </tbody>
+</table>
+
+
+## How to use your data
+
+<CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="logs"
+    title="Access logs"
+  >
+Use AWS Lambda to send CloudWatch Apprunner logs to New Relic. Learn how to [install and configure the Cloudwatch logs Lambda function ](/docs/logs/forward-logs/aws-lambda-sending-cloudwatch-logs/#install-function).
+  </Collapser>
+  <CollapserGroup>
+  <Collapser
+    className="freq-link"
+    id="alerts"
+    title="Create alerts"
+  >
+There are countless things alerts could be set up for. For example, an alert can be set up to notify relevant parties of critical or fatal errors:
+
+1. Click the ellipses on the dashboards's frame and choose **Create alert condition** option
+2. In the pane that appears edit the nrql query to: ADD QUERY HERE
```

Comment:
What does the query need to be updated to so it alerts on "critical or fatal errors"?